packages/cli/src/commands/bisect/command.ts

```diff
@@ -0,0 +1,69 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const bisectCommand: Command = {
+  name: 'bisect',
+  description: 'Bisect the current project interactively',
```

Comment:
End the desc with a period.

```suggestion
  description: 'Bisect the current project interactively.',
```

Reply:
It's not currently. Is this an update you'd like to see for all descriptions?